lib/Backend/LowerMDSharedSimd128.cpp

```diff
@@ -915,14 +915,13 @@ IR::Instr* LowererMD::Simd128CanonicalizeToBools(IR::Instr* instr, const Js::OpC
 
 void LowererMD::EmitSimdConversion(IR::Instr *instr, IR::JnHelperMethod helper)
 {
-    IR::Opnd* dst = instr->GetDst();
```

Comment:
I think I would prefer if the function doing the Unlink is the same freeing the instr.
It makes it more clear that the opnd cannot be used in between without looking at the implementation.
I recommend changing the signature of this function to `EmitSimdConversion(IR::Opnd* dst, IR::Opnd* src, IR::JnHelperMethod helper)` and call unlink at the call site

Reply:
we still need to pass the instr as an argument to EmitSimdConversion because of its use in loading helper memref arguments. 

Shall we move removeInstr() to be inside EmitSimdConversion?

Reply:
You can move the remove inside this function if you prefer.
The instr is only used as a reference to where to insert the argument.
Usually (well sometimes) we call it `insertBeforeInstr` to mean that we don't actually operate on that instr, but we need to insert new ones before it.

Reply:
That makes sense. Thanks @Cellule . Please take a look.